NVDSuppressions.xml

```diff
@@ -34,6 +34,37 @@ See https://nvd.nist.gov/vuln/detail/CVE-2005-2224 and https://www.securityfocus
    ]]>    </notes>
     <cve>CVE-2008-0333</cve>
   </suppress>
-</suppressions>
 
-  
\ No newline at end of file
+  <suppress>
+    <notes>
+      <![CDATA[
+    file name: Serilog.Sinks.File:4.1.0
+      8117-softmagic.c in file before 5.21 does not properly limit recursion, which allows remote attackers to cause a denial of service (CPU consumption or crash) via unspecified vectors.
+      9652-The mconvert function in softmagic.c in file before 5.21, as used in the Fileinfo component in PHP before 5.4.37, 5.5.x before 5.5.21, and 5.6.x before 5.6.5, does not properly handle a certain string-length field during a copy of a truncated version of a Pascal string, which might allow remote attackers to cause a denial of service (out-of-bounds memory access and application crash) via a crafted file.
+    This is a false positive. These two CVEs references a C library softmagic.c which is not used by FSS
```

Comment:
This is the Exchange Set Service, not FSS

Reply:
Resolved by renaming as ESS.